grails-datastore-gorm-hibernate5/build.gradle

```diff
@@ -20,7 +20,7 @@ dependencies {
     }
     runtimeOnly "jakarta.el:jakarta.el-api:$jakartaElVersion"
     runtimeOnly "jakarta.xml.bind:jakarta.xml.bind-api:$jakartaXmlBindVersion"
-    runtimeOnly "com.sun.xml.bind:jaxb-impl:4.0.5"
+    runtimeOnly "org.glassfish.jaxb:jaxb-runtime:$jakartaXmlBindImpVersion"
```

Comment:
Why is `jaxb` even necessary at runtime for `org.grails:grails-datastore-gorm-hibernate5`?

Reply:
It was historically there: https://mvnrepository.com/artifact/org.grails/grails-datastore-gorm-hibernate5/8.1.0

But I did not find it currently used in any of the github.com/grails repos:  
https://github.com/search?q=org%3Agrails+org.glassfish.jaxb&type=code
https://github.com/search?q=org%3Agrails+com.sun.istack&type=code
https://github.com/search?q=org%3Agrails+com.sun.xml.txw2&type=code

Do you know if it is safe to remove?

Reply:
These dependencies were added in commit https://github.com/grails/gorm-hibernate5/commit/a8ad0c728f238394760ab02f118a771f54d04d8d with the commit message "Add dependencies required for Java 11".

It seems JAXB previously included in the JDK but removed in Java 11.
However, that does not answer my initial question; why is it needed in `org.grails:grails-datastore-gorm-hibernate5`.

Is there any marshalling/unmarshalling of XML in this module?

Reply:
**hibernate-core-jakarta**
```
|    +--- org.hibernate:hibernate-core-jakarta:5.6.15.Final
|    |    \--- org.glassfish.jaxb:jaxb-runtime:3.0.0 -> 4.0.5
|    |         \--- org.glassfish.jaxb:jaxb-core:4.0.5
|    |              +--- jakarta.xml.bind:jakarta.xml.bind-api:4.0.2 (*)
|    |              +--- jakarta.activation:jakarta.activation-api:2.1.3
|    |              +--- org.eclipse.angus:angus-activation:2.0.2
|    |              |    \--- jakarta.activation:jakarta.activation-api:2.1.3
|    |              +--- org.glassfish.jaxb:txw2:4.0.5
|    |              \--- com.sun.istack:istack-commons-runtime:4.1.2
```

if it isn't needed for compilation, remove it.

Reply:
It is not used in gorm-hibernate5 or in any current code within the github.com/grails repos.  I am creating another PR to pull it out.

Reply:
https://github.com/grails/gorm-hibernate5/pull/892

Reply:
Also included same changes in https://github.com/grails/grails-core/pull/13661